src/css/index.css

```diff
@@ -6,69 +6,63 @@
   --button-color-hover: #6f6f6f;
   --border-color: #c2c2c2;
   --text-open-sans: "Open Sans", Arial, Helvetica, sans-serif;
-  --text-roboto: "Roboto", Arial, Helvetica, sans-serif; }
+  --text-roboto: "Roboto", Arial, Helvetica, sans-serif;
+}
 
 /******* Normalize *******/
 * {
   margin: 0;
   padding: 0;
-  box-sizing: border-box; }
+  box-sizing: border-box;
+  font-family: var(--text-open-sans);
+  color: var(--text-color);
+}
+
+* a {
+  text-decoration: none;
+}
 
 html {
-  height: 100%; }
+  height: 100%;
+}
 
 body {
   max-width: 60em;
-  min-height: 100%; }
+  min-height: 100%;
+}
 
 /****** Typography ******/
 h1 {
   font-family: var(--text-open-sans);
-  font-size: 1.6rem; }
+  font-size: 1.6rem;
+}
 
 h2 {
-  font-size: 1.4rem; }
+  font-size: 1.4rem;
+}
 
 h3 {
-  font-size: 1.2rem; }
+  font-size: 1.2rem;
+}
 
 /* This is the main content */
 .fullContainer {
   height: 100%;
-  width: 100%; }
+  width: 100%;
+}
 
 /* This is the map part of the content */
 .mapContainer {
   background-color: white;
-  padding: .5em; 
-}
-
-.loadingIndicator {
```

Comment:
@PepperAddict would you salvage @sgluzins styling by moving it over to scss? I missed that on #184 somehow.

Reply:
sure thing.